src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/create_index_pattern_wizard.js

```diff
@@ -165,7 +166,7 @@ uiModules.get('apps/management')
 
   this.fetchExistingIndices = () => {
     this.isFetchingExistingIndices = true;
-    const allExistingLocalAndRemoteIndicesPattern = '*,*:*';
+    const allExistingLocalAndRemoteIndicesPattern = '*';
```

Comment:
Does this variable name still make sense? If I recall correctly, the `*,*:*` pattern is a cross-cluster search pattern, which is what the "remote" indices part of the variable name refers to. So maybe this variable name now needs to be just `allExistingIndices`?